package.json

```diff
@@ -86,7 +86,7 @@
         "bazel:lint-fix": "yarn bazel:format --lint=fix",
         "format": "git-clang-format",
         "format-all": "clang-format --glob='{internal/**/,examples/**/}*.{js,ts}' -i",
-        "skydoc": "bazel build //docs && unzip -o -d docs dist/bin/docs/docs-skydoc.zip",
+        "skydoc": "bazel build //docs && cp -f dist/bin/docs/*.md docs",
```

Comment:
`rm -rf docs` before `cp`?

Reply:
not that simple, some of the markdown files in the docs/ folder are generated but some are not.